events/triggers/passwd

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+signal="${1}"
+
+# Try to load Umbrel OS version info to detect Umbrel OS
+[[ -f "/etc/default/umbrel" ]] && source "/etc/default/umbrel"
+
+
+# Only change password on Umbrel OS
+if [[ ! -z "${UMBREL_OS:-}" ]]; then
+    # Script runs as root, we need to set the username
+    echo ${1} | passwd umbrel --stdin
```

Comment:
Did you mean to write `cat "$signal" | passwd umbrel --stdin`?

I don't know much about Umbrel but it seems to me you're storing this inside the file not in the file name?

Reply:
I already said that I will change it.